### PR TITLE
bugfix/AB#110

### PR DESCRIPTION
**Pull request by DerCed**
## Description
Added Entity title to EntityDialog Title
AB#110

## Motivation and Context
Request by Product Owner

## How has this been tested?
Local Browser

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: webclient/app/src/localization/translations-de-DE.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ const translationsDeDE = {
     "entityDialog.noRelations": "Keine Relationen angelegt.",
     "entityDialog.noFields": "Keine Felder angelegt.",
     "entityDialog.domainIsLoading": "Domain wird geladen.",
-    "entityDialog.editDomain": "Domain bearbeiten.",
+    "entityDialog.editDomain": "{{entityName}} bearbeiten.",
```
Comment (witchpou) [inline webclient/app/src/localization/translations-de-DE.js:46]: its better to call it entity than domain
```suggestion
    "entityDialog.editEntity": "{{entityName}} bearbeiten.",
```

File: webclient/app/src/localization/translations-en-EN.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ const translationsEnEN = {
     "entityDialog.noRelations": "No relations.",
     "entityDialog.noFields": "No fields.",
     "entityDialog.domainIsLoading": "Domain is loading.",
-    "entityDialog.editDomain": "Edit domain.",
+    "entityDialog.editDomain": "Edit {{entityName}}.",
```
Comment (witchpou) [inline webclient/app/src/localization/translations-en-EN.js:46]:
```suggestion
    "entityDialog.editEntity": "Edit {{entityName}}.",
```

File: webclient/app/src/app/commons/entityDialog/EntityDialog.jsx [hunk 3]
```diff
@@ -260,7 +260,13 @@ function EntityDialog(props) {
     return (
         <Dialog open={!!entityId} maxWidth={"xl"} fullWidth>
             <DialogTitle className={entityEditorStyles.dialogHeaderBar}>
-                <Typography noWrap variant={"h6"} component={"p"}>{t("entityDialog.editDomain")}</Typography>
+                <Typography
+                    noWrap
+                    variant={"h6"}
+                    component={"p"}
+                >
+                    {t("entityDialog.editDomain", {entityName: entity.name})}
```
Comment (witchpou) [inline webclient/app/src/app/commons/entityDialog/EntityDialog.jsx:268]:
```suggestion
                    {t("entityDialog.editEntity", {entityName: entity.name})}
```

**Review by witchpou — CHANGES_REQUESTED** [review]
Domain -> Entity

**Comment by DerCed** [comment]
> Domain -> Entity

Which Format do you want exactly?
Shall I rename all Domain references to Entity?

**Review by witchpou — APPROVED** [review]

**Pull request merged**